README.md

```diff
@@ -15,6 +15,11 @@ Here is some example configuration that demonstrates the structure, using all re
         versions:
```

Comment:
This `versions` no longer exists in your new structure, right?

Reply:
Oh, but I guess it makes sense to keep this until all apps move to this new format

Reply:
Well apps can choose from two ways of defining their API docs:
* Flat - most apps will use this since they are using just one API docs
* Nested - if app uses multiple API docs they can define them in nested structure

Reply:
Ah okay, that makes sense!